fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeViewModel.java

```diff
@@ -47,8 +47,9 @@
     implements CameraPermissionViewModel, QRCodeScanningViewModel {
 
   public static final String TAG = HomeViewModel.class.getSimpleName();
-  public static final String SCAN = "SCAN";
-  public static final String REQUEST_CAMERA_PERMISSION = "REQUEST_CAMERA_PERMISSION";
+
+  public enum HomeViewAction {SCAN, REQUEST_CAMERA_PERMISSION}
```

Comment:
Actually, I just realized that this Action is also used in `LaoDetailViewModel`.

So, does it make sense to keep it inside `HomeViewModel` ?
Just asking, this is really nitpicking.